engine/src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -581,8 +582,10 @@ class WorkflowActor(workflowToStart: WorkflowToStart,
 
     def deleteFiles() = {
       val rootWorkflowId = data.workflowDescriptor.get.rootWorkflowId
+      val rootWorkflowRoots = data.initializationData.data.values.collect({case Some(i: StandardInitializationData) => i.workflowPaths.workflowRoot}).toSet[Path]
       val deleteActor = context.actorOf(DeleteWorkflowFilesActor.props(
         rootWorkflowId = rootWorkflowId,
+        rootWorkflowRoots = rootWorkflowRoots,
```

Comment:
Mild preference: is there a way to name this that more clearly suggests its purpose, such as `rootWorkflowProtectedPaths`? Especially since it's only used in deletion, seemingly.

Reply:
Yes, I didn't like that name either. But I don't think "Protected" is the right thing to add because these aren't the paths that protected-- everything _else_ is protected. I changed the variable to rootWorkflowRootPaths to try to clarify what the second "root" refers to.